examples/app.py

```diff
@@ -49,17 +65,27 @@
 from __future__ import absolute_import, print_function
 
 from flask import Flask, render_template
+
 from flask_babelex import Babel
+
 from flask_cli import FlaskCLI
+
 from flask_mail import Mail
+
 from flask_security import current_user
-from invenio_db import InvenioDB
 
 from invenio_accounts import InvenioAccounts
 
+from invenio_assets import InvenioAssets
+
+from invenio_db import InvenioDB
+
+from invenio_theme import InvenioTheme
+
 # Create Flask application
 app = Flask(__name__)
 app.config.update(
+    ACCOUNTS_BASE_TEMPLATE='invenio_theme/page.html',
```

Comment:
I don't think you have to set this variable. invenio-accounts will default to ``BASE_TEMPLATE`` if it's not set, and invenio-theme will set ``BASE_TEMPLATE``